deepomatic/api/resources/network.py

```diff
@@ -66,6 +68,12 @@ def inference(self, convert_to_numpy=True, return_task=False, **kwargs):
         else:
             return result
 
+    def create(self, *args, **kwargs):
+        # No retry on network create by default as this is an heavy request
+        kwargs['http_retry'] = kwargs.get('http_retry')
```

Comment:
```suggestion
        kwargs['http_retry'] = kwargs.get('http_retry', None)
```
Explicitly show we emit `None` by default makes it more readable IMO

Reply:
I would agree if it wasn't `get()` which is one of the most common python function. I committed the suggestion though.